keyboards/zinc/keymaps/toshi0383/keymap.c

```diff
@@ -0,0 +1,130 @@
+   * |------+------+------+------+------+------|             |------+------+------+------+------+------|
+   * |      |      |      |      | GUI  |Shift |             |Shift |------|      |      |      |      |
+   * `-----------------------------------------'             `-----------------------------------------'
+   */
+  [_RAISE] = LAYOUT_ortho_4x12( \
+  _______, KC_1,    KC_2,    KC_3,    KC_4,    KC_5,           KC_6,    KC_7,    KC_8,    KC_9,    KC_0,  _______, \
+  _______, KC_COLN, KC_SCLN, KC_DQT, KC_QUOT, KC_BSLS,         KC_LEFT, KC_DOWN, KC_UP, KC_RGHT, _______, _______, \
+  _______, _______, _______, _______, _______, _______,        ADJUST, _______, _______, _______, _______, _______, \
+  _______, _______, _______, _______, KC_LGUI, KC_LSFT,        KC_LSFT, _______, _______, _______, _______, _______ \
+  ),
+
+  /* Adjust (Lower + Raise)
+   * ,-----------------------------------------.             ,-----------------------------------------.
+   * |      |      |      |      |      |      |             |      |      |      |      |      |      |
+   * |------+------+------+------+------+------|             |------+------+------+------+------+------|
+   * |      |      |      |      |      |      |             |      |      |      |      |      |      |
+   * |------+------+------+------+------+------|             |------+------+------+------+------+------|
+   * |      |      |      |      |      |      |             |      |      |      |      |      |      |
+   * |------+------+------+------+------+------|             |------+------+------+------+------+------|
+   * |      |      |      |      |      |RESET |             |------|      |      |      |      |      |
+   * `-----------------------------------------'             `-----------------------------------------'
+   */
+    [_ADJUST] =  LAYOUT_ortho_4x12( \
+      _______, _______, _______, _______, _______, _______,  _______, _______, _______, _______, _______, _______, \
+      _______, _______, _______, _______, _______, _______,  _______, _______, _______, _______, _______, _______, \
+      _______, _______, _______, _______, _______, _______,  _______, _______, _______, _______, _______, _______, \
+      _______, _______, _______, _______, _______, RESET,  _______, _______, _______, _______, _______, _______ \
+      ),
+
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case LOWER:
+      if (record->event.pressed) {
+        layer_on(_LOWER);
+      } else {
+        layer_off(_LOWER);
+      }
+      return false;
+      break;
+
+    case RAISE:
+      if (record->event.pressed) {
+        layer_on(_RAISE);
+      } else {
+        layer_off(_RAISE);
+      }
+      return false;
+      break;
+
+    case ADJUST:
+      if (record->event.pressed) {
+        layer_on(_ADJUST);
+      } else {
+        layer_off(_ADJUST);
+      }
+      return false;
+      break;
+
+  }
+
+  return true;
+}
```

Comment:
This whole function could be removed and the `custom_keycodes` enum replaced with simple `MO()` aliases, as you've done for your reviung39 keymap.

Reply:
Thank you!
Done in https://github.com/qmk/qmk_firmware/pull/7141/commits/c902ae2287bd758111c2b49499c12b3906cc4ff1